src/tests/gssapi/t_s4u.c

```diff
@@ -305,6 +307,32 @@ main(int argc, char *argv[])
         fprintf(stderr, "\n");
     }
 
+    if (delegated_cred_handle != GSS_C_NO_CREDENTIAL) {
+        /* Inquire impersonator status */
+        major = gss_inquire_cred_by_oid(&minor, user_cred_handle, &inq_oid,
+                                        &bufset);
+        check_gsserr("gss_inquire_cred_by_oid", major, minor);
+
+        if (bufset->count != 1 || bufset->elements[0].length != sizeof(int))
+            errout("gss_inquire_cred_by_oid() returned invalid buffer set");
+
+        if (*((int *)bufset->elements[0].value) != 1)
```

Comment:
This could bus error on a platform with strict alignment requirements.  (Same for the similar condition below.)